_src-js/lib/bound-properties.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import videojs from 'video.js';
+import shortid from 'shortid';
+
+const Form = React.createClass({
+  getInitialState() {
+    return {
```

Comment:
Yeah, it's a little annoying that they've deprecated some stuff (like mixins) and said that they suggest moving to ES2015 components like this....but then don't actually document very much in that way. There is one [quick blurb on them](https://facebook.github.io/react/docs/reusable-components.html#es6-classes), though.